matRad_calcCubes.m

```diff
@@ -34,14 +34,14 @@
     scenNum = 1;
 end
 
-% consider VOI priorities
-cst  = matRad_setOverlapPriorities(cst);
-
 resultGUI.w = w;
 
 % calc dose and reshape from 1D vector to 2D array
 resultGUI.physicalDose = reshape(dij.physicalDose{scenNum}*resultGUI.w,dij.dimensions);
 
+% consider VOI priorities
+[cst,resultGUI.overlapCube]  = matRad_setOverlapPriorities(cst,size(resultGUI.physicalDose));
```

Comment:
Can you change this to 

matRad_setOverlapPriorities(cst,dij.dimensions);

I argue this is more intuitive for outsideres!